### ISSUE TITLE
Deploy command 

### ISSUE THREAD

**Issue by it-can** [bugsnag/bugsnag-laravel#369]
In the file https://github.com/bugsnag/bugsnag-laravel/blob/master/src/Commands/DeployCommand.php  I see that ```Bugsnag\Bugsnag\Utils``` is referenced, but this does not exist in the codebase... 

Also when app_version is not set, the deploy command seems to run fine. It will log to the errorlog that the app_version is not set. I think it should display an error on screen.

**Comment by mattdyoung** [comment]
Hi @it-can 

`Bugsnag\Bugsnag\Utils` exists in the bugsnag-php dependency:
https://github.com/bugsnag/bugsnag-php/blob/master/src/Utils.php

How are you running the deploy command? Doing a `php artisan bugsnag:deploy` as described in the [documentation](https://docs.bugsnag.com/platforms/php/laravel/#tracking-releases) displays the error on screen:
```
$ php artisan bugsnag:deploy
Bugsnag Warning: App version is not set. Unable to send build report.
```

**Comment by it-can** [comment]
shouldn't it be ```use Bugsnag\Utils;``` ?

**Comment by abigail-bugsnag** [comment]
Thanks @it-can, I believe you are correct. We will take a look at this.

**Comment by mattdyoung** [comment]
Fix released in [v2.18.0](https://github.com/bugsnag/bugsnag-laravel/releases/tag/v2.18.0)
